lib/js/components/Headers/SectionHeader/SectionHeader.vue

```diff
@@ -79,9 +85,75 @@
 @import '../../../../styles/settings/media-queries';
 @import '../../../../styles/settings/spacings';
 
+$icon-colors: (
+	'accent': (
+		'color': $color-accent-icon,
+		'colorHover': $color-accent-icon-hovered,
+	),
+	'danger': (
+		'color': $color-danger-icon,
+		'colorHover': $color-danger-icon-hovered,
+	),
+	'default': (
+		'color': $color-default-icon,
+		'colorHover': $color-default-icon,
+	),
+	'fail': (
+		'color': $color-fail-icon,
+		'colorHover': $color-fail-icon-hovered,
+	),
+	'info': (
+		'color': $color-info-icon,
+		'colorHover': $color-info-icon-hovered,
+	),
+	'inverted': (
+		'color': $color-inverted-icon,
+		'colorHover': $color-inverted-icon,
+	),
+	'neutral': (
+		'color': $color-neutral-icon,
+		'colorHover': $color-neutral-icon-hovered,
+	),
+	'neutralStrong': (
+		'color': $color-neutral-icon-strong,
+		'colorHover': $color-neutral-icon-strong-hovered,
+	),
+	'neutralWeak': (
+		'color': $color-neutral-icon-weak,
+		'colorHover': $color-neutral-icon-weak-hovered,
+	),
+	'primary': (
+		'color': $color-primary-icon,
+		'colorHover': $color-primary-icon-hovered,
+	),
+	'primaryWeak': (
+		'color': $color-primary-icon-weak,
+		'colorHover': $color-primary-icon-weak-hovered,
+	),
+	'success': (
+		'color': $color-success-icon,
+		'colorHover': $color-success-icon-hovered,
+	),
+	'warning': (
+		'color': $color-warning-icon,
+		'colorHover': $color-warning-icon-hovered,
+	),
+);
+
 .sectionHeader {
 	$root: &;
 
+	@each $color-name, $color-map in $icon-colors {
+		&__icon.-#{$color-name},
+		&.-size-xx-small &__icon.-#{$color-name} {
```

Comment:
just __icon isn't enough

Reply:
Nope, because of different default color for xx-small